File: fizzbuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        for i in range(1,n+1) :
            mod3 = i % 3
            mod5 = i % 5
            if (mod3 == 0) and (mod5 == 0) :
                answer.append("FizzBuzz")
                continue
            if mod3 == 0 :
                answer.append("Fizz")
                continue
            if mod5 == 0 :
                answer.append("Buzz")
                continue
            answer.append(str(i))
        return answer

File: fizzbuzz.go
func fizzBuzz(n int) []string {
	answer := make([]string, n)
	for i := 1; i <= n; i++ {
    	mod3 := i % 3
			mod5 := i % 5
			if mod3 == 0 && mod5 == 0 {
					answer[i-1] = "FizzBuzz"
					continue
			}
			if mod3 == 0 {
					answer[i-1] = "Fizz"
					continue
			}
			if mod5 == 0 {
					answer[i-1] = "Buzz"
					continue
			}
			answer[i-1] = strconv.Itoa(i)
	}
	return answer
}
